src/_shared/scss/_adverts.scss

```diff
@@ -66,7 +66,8 @@
         > .adverts__header {
             box-sizing: border-box;
             padding: $gs-baseline $gs-gutter / 5 * 2;
-            width: 160px;
+            // Extra 1px to line up with borders on sections above / below
+            width: 160px + $gs-gutter/2 + 1px;
```

Comment:
Not a biggie but it would be nice to have consistent formatting e.g. `160px + $gs-gutter / 2 + 1px`